book/src/development/infrastructure/changelog_update.md

```diff
@@ -110,6 +112,9 @@ grep -rB1 "pub $LINT_NAME" .
 The version shown should match the version of the release the changelog is 
 written for. If not, update the version to the changelog version.
 
+Newly created lints will have `#[clippy::version = "nightly"]` and be handled
```

Comment:
Nit: "dev" could be more appropriate, as code merged into the Clippy repository doesn't end up immediately into the next "nightly" release as distributed through rustup. I'm afraid using "nightly" could lead to reports that the lint isn't available in nightly, whereas "dev" makes it clearer that it may or may not be present.

Reply:
And right after this comment, I open [this Zulip thread](https://rust-lang.zulipchat.com/#narrow/channel/257328-clippy/topic/.E2.9C.94.20Unknown.20clippy.20lints/with/561679204) which makes my point.